src/lib.rs

```diff
@@ -474,10 +472,11 @@ mod test {
     }
 
     #[cfg(target_os = "linux")]
+    #[cfg_attr(miri, ignore)]
```

Comment:
would it be better to have CI be noisy here? This may help hide underlying issues.

Reply:
> would it be better to have CI be noisy here? This may help hide underlying issues.

I marked this test as ignored because it accesses the filesystem, which miri can't do (sandboxing issue). This is not related to the code of mownstr but to the code of the test itself, so I don't think it is a significant issue to report.

> I would suggest this be added to miri test:

Yes, I added the "with" directive in a further commit: https://github.com/pchampin/mownstr/commit/05c3350922e0db26388c16fc7aaab7717fea4804

Reply:
Could this instead be done on the locations where `m0` and `m1` are defined & invoked so that there's still miri coverage? That may be overzealous but this would give us miri coverage of https://github.com/pchampin/mownstr/blob/815c793bf80ed0364a64aba3f40a4f61dae55e9a/src/lib.rs#L491-L498

Reply:
fair enough, I will proceed otherwise

Reply:
solved by d27f233